docs/Docker.md

```diff
@@ -1,25 +1,82 @@
 # Using Docker
 
+## Server Initialization
+This was last tested on Ubuntu 24.04 x86
+Note: For this document, macadmins/crypt-server and crypt-server are used interchangeably. macadmins/crypt-server references the downloadedy option and crypt-server represents the local build option. 
```

Comment:
Stick to the built image please.

Reply:
Will revise to indicate using the built image. 

Is it possible to get the image rebuilt more frequently to incorporate upstream library changes?

Reply:
The latest image is built after every commit to the master branch. That of course does not stop you from building your own.